packages/typespec-ts/test/modularIntegration/generated/client/naming/src/api/operations.ts

```diff
@@ -0,0 +1,246 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import {
+  ClientNameModel,
+  LanguageClientNameModel,
+  ClientNameAndJsonEncodedNameModel,
+} from "../models/models.js";
+import {
+  HeaderRequest204Response,
+  HeaderResponse204Response,
+  NamingContext as Client,
+  Operation204Response,
+  Parameter204Response,
+  PropertyClient204Response,
+  PropertyCompatibleWithEncodedName204Response,
+  PropertyLanguage204Response,
+} from "../rest/index.js";
+import {
+  StreamableMethod,
+  operationOptionsToRequestParameters,
+  createRestError,
+  OperationOptions,
+} from "@azure-rest/core-client";
+import {
+  ParameterOptions,
+  ClientRequestOptions,
+  LanguageOptions,
+  CompatibleWithEncodedNameOptions,
+  RequestOptions,
+  ResponseOptions,
+} from "../models/options.js";
+
+export function _operationSend(
+  context: Client,
+  options: OperationOptions = { requestOptions: {} },
+): StreamableMethod<Operation204Response> {
+  return context
+    .path("/client/naming/operation")
+    .post({ ...operationOptionsToRequestParameters(options) });
+}
+
+export async function _operationDeserialize(
+  result: Operation204Response,
+): Promise<void> {
+  if (result.status !== "204") {
+    throw createRestError(result);
+  }
+
+  return;
+}
+
+export async function operation(
+  context: Client,
+  options: OperationOptions = { requestOptions: {} },
+): Promise<void> {
+  const result = await _operationSend(context, options);
+  return _operationDeserialize(result);
+}
+
+export function _parameterSend(
+  context: Client,
+  defaultName: string,
```

Comment:
why not clientName?

Reply:
oh I see that, thanks for pointing it out